app/routes/first-time/benefits.js

```diff
@@ -0,0 +1,37 @@
+const benefitValidator = require('../../services/validators/eligibility/benefit-validator')
```

Comment:
if refactoring eligibility -> first-time, we should maybe do the same with the validators?